Wikipedia/Code/WebViewController.m

```diff
@@ -409,6 +417,11 @@ - (void)scrollToSection:(MWKSection*)section animated:(BOOL)animated {
     [self scrollToFragment:section.anchor animated:animated];
 }
 
+- (void)accessibilityCursorToSection:(MWKSection*)section {
+    NSString *js = [NSString stringWithFormat:@"focus_element = document.getElementById('%@'); other_element = document.body;  other_element.setAttribute('tabindex', 0); focus_element.setAttribute('tabindex', 0); other_element.focus(); focus_element.focus();", section.anchor];
```

Comment:
@montehurd we can refactor this into a JS script which we invoke via the JS context. I wouldn't worry about breaking into multiple lines.

@hhanke we also have some JS functions which are included in the page.  the process for modifying those is documented in `docs/web-dev.md` if you're feeling brave. If not, we can refactor it after this is merged